Dockerfile

```diff
@@ -20,11 +24,13 @@ COPY package.json package-lock.json env.sh tsconfig.json $HOME/
 COPY .npmrcs/$NPMLOCATION .npmrc
 RUN chmod a+w "$HOME/package-lock.json"
 
+
 # USER doesn't impact on COPY
 USER 1001
 
 WORKDIR $HOME
-RUN echo "Using npm location: $NPMLOCATION" && \
+RUN echo "Building from commit $GITCOMMIT" && \
```

Comment:
Please make it more universal, when the image is built locally.

Reply:
Do you only want the message to say something like "building locally" or do you want to include the Git commit there? The latter would require copying the `.git` directory inside the container which we currently avoid.

Reply:
I've adjusted the message.